app/templates/createquiz.html

```diff
@@ -5,39 +5,12 @@
 {% block content %}
 <div class="create-class-wrapper">
     <div class="create-container">
+
         <form action="" method="post" novalidate class="create-form">
```

Comment:
Can't this be done on the dashboard itself? Then can just post request to the createquiz link. Same for createclass.